src/screens/TokenDetail.js

```diff
@@ -21,85 +21,66 @@ import { TX_COUNT } from '../constants';
 class TokenDetail extends React.Component {
   /**
    * token {Object} selected token data
-   * successMessage {string} success message to show
    * errorMessage {string} error message to show
-   * totalSupply {number} Token total supply
-   * canMint {boolean} If this token can still be minted
-   * canMelt {boolean} If this token can still be melted
-   * paramUID {string} UID of token in the URL
    * transactions {Array} Array of transactions for the token
    */
   state = {
```

Comment:
We should have at least all keys that will be eventually created here.

Reply:
All those keys will be inside `token` object now